### PR TITLE
chore(docs): remove duplicate sentence

### PR DESCRIPTION
**Pull request by alexiscarbillet**
## Changes

I change a line in the documentation "Getting started". 

## Context
It is disturbing to have two lines in the row with the same sentence. The only difference was a ":".

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [x] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by Thompson1985 — APPROVED** [review]

File: docs/usage/getting-started/use-cases.md [hunk 1]
```diff
@@ -199,7 +199,7 @@ Here's how it works:
 
 #### Benefits of using Dependency Dashboard Approval
 
-Benefits of using Dependency Dashboard Approval:
+There are different benefits for using Dependency Dashboard Approval:
```
Comment (HonkingGoose) [inline docs/usage/getting-started/use-cases.md:202]:
```suggestion
```

Alternate option: drop the platform sentence, and rely on the heading.

File: docs/usage/getting-started/use-cases.md [hunk 2]
```diff
@@ -199,7 +199,6 @@ Here's how it works:
 
 #### Benefits of using Dependency Dashboard Approval
 
-Benefits of using Dependency Dashboard Approval:
```
Comment (HonkingGoose) [inline docs/usage/getting-started/use-cases.md:202]:
```suggestion
```

**Review by HonkingGoose — APPROVED** [review]
I'm happy with the content change.

But _before_ we merge, we should change the PR title. How about this?

`docs(use cases): remove duplicate sentence`

**Review by rarkins — APPROVED** [review]

**Comment by HonkingGoose** [comment]
@alexiscarbillet congratulations on getting your first PR for this project merged. 🥳

**Pull request merged**